manim/animation/animation.py

```diff
@@ -49,6 +49,13 @@ class Animation:
 
         For example ``rate_func(0.5)`` is the proportion of the animation that is done
         after half of the animations run time.
+
+
+    reverse_rate_function
+        Whether the animation need to be played backwards.
+
+        Setting `reverse_rate_function` doesn't have any effect on `remover` and `introducer`.
+            If you want your animation to be a introducer or remover. You must set it manually.
```

Comment:
Minor suggestion regarding wording.

```suggestion
        Reverses the rate function of the animation. Setting ``reverse_rate_function``
        does not have any effect on ``remover`` or ``introducer``. These need to be
        set explicitly if an introducer-animation should be turned into a remover one
        and vice versa.
```

Reply:
LGTM!